xbmc/music/Song.cpp

```diff
@@ -38,7 +39,33 @@ CSong::CSong(CFileItem& item)
   std::vector<std::string> musicBrainArtistHints = tag.GetMusicBrainzArtistHints();
   strArtistDesc = tag.GetArtistString();
   if (!tag.GetMusicBrainzArtistID().empty())
-  { // have musicbrainz artist info, so use it
+  { // Have musicbrainz artist info, so use it
+    // Establish tag consistency - do the number of musicbrainz ids and number of names in hints or artist match
```

Comment:
Might make sense to create a music Utility class and just call that, the code seems to be quite similar